[DataGridPremium] Server-side data source with row grouping (#13826)
mui/mui-x: docs/data/data-grid/server-side-data/tree-data.md

};
```

> Apart from the other parameters like `filterModel`, `sortModel`, and `paginationModel`, the `getRows()` callback receives a `groupKeys` parameter. This corresponds to the keys provided for each nested level in `getGroupKey()`. Use `groupKeys` on the server to fetch the rows for each nested level.

Author: ```suggestion
Like the other parameters such as `filterModel`, `sortModel`, and `paginationModel`, the `getRows()` callback receives a `groupKeys` parameter that corresponds to the keys provided for each nested level in `getGroupKey()`.
Use `groupKeys` on the server to extract the rows for a given nested level.
```
